scipy/differentiate/tests/test_differentiate.py

```diff
@@ -12,15 +12,15 @@
 from scipy.differentiate import derivative, jacobian, hessian
 from scipy.differentiate._differentiate import _EERRORINCREASE
 
+pytestmark = [array_api_compatible, pytest.mark.usefixtures("skip_xp_backends")]
 
-@array_api_compatible
-@pytest.mark.usefixtures("skip_xp_backends")
-@pytest.mark.skip_xp_backends('array_api_strict',
-                              reason='Currently uses fancy indexing assignment.')
-@pytest.mark.skip_xp_backends('jax.numpy',
-                              reason='JAX arrays do not support item assignment.')
-@pytest.mark.skip_xp_backends('cupy',
-                              reason='cupy/cupy#8391')
+array_api_strict_skip_reason = 'Array API does not support fancy indexing assignment.'
+jax_skip_reason = 'JAX arrays do not support item assignment.'
+torch_skip_reason = 'Torch does not have `take_slong_axis`.'
```

Comment:
I think this is unused now 
```suggestion
```

Reply:
Ah, yes. OK, if everything passes, I'll merge this with `[skip ci]`